core/db/migrate/20130326175857_add_stock_location_to_rma.rb

```diff
@@ -1,4 +1,4 @@
-class AddStockLocationToRma < ActiveRecord::Migration[4.2]
+class AddStockLocationToRMA < ActiveRecord::Migration[4.2]
```

Comment:
Due to
> NameError: uninitialized constant AddStockLocationToRMA

while running migrations, I was supposed to change name of class using upper case.
I didn't have to rollback any migrations because file name is the same.